Zend/zend_API.c

```diff
@@ -439,6 +439,11 @@ ZEND_API ZEND_COLD void zend_argument_value_error(uint32_t arg_num, const char *
 }
 /* }}} */
 
+ZEND_API ZEND_COLD void zend_argument_cannot_be_empty_error(uint32_t arg_num)
```

Comment:
The function name now no longer matches the message.

Reply:
Updated in-place in the last commit, which changes the wording from "cannot" to "must not"